### DISCUSSION TITLE
A new version of TShock

### DISCUSSION THREAD

**Discussion by ghost** [Pryaxis/TShock#2983] [Ideas]
I think that Pryaxis should make a new version of TShock (a fork) and modify it to where the server hosting mod is more like an RPG with :  `SSC forced on, no max hp, a level system, towns, multiple worlds (must be generated on first entry), and more events (events can be combined).`

**Comment by punchready** [comment]
Your ambitions are appreciated, but that is not the purpose of TShock or Pryaxis. TShock is a base framework enabling *others* to make these creations without locking them into a particular design. To enjoy a Terraria RPG server you can play on one of the already available servers on https://terraria-servers.com/.
